python/lsst/ts/nightreport/create_tables.py

```diff
@@ -0,0 +1,70 @@
+__all__ = [
```

Comment:
Please make sure that all Python files everywhere in the project (including the doc and tests directories) have the common TSSW header.